README.rst

```diff
@@ -65,6 +65,13 @@ Features
 
 -----
 
+=============
+Prerequisites
+=============
+
+* Python >= 3.4
+* pip >= 8.0
```

Comment:
also it appears your commits have no authorship. you should link your github account by using git config (google the commands, theyre easy to find)